eland/dataframe.py

```diff
@@ -844,32 +848,31 @@ def info(
 
         lines = [str(type(self)), self._index_summary()]
 
-        if len(self.columns) == 0:
+        cols: pd.Index = self.columns
```

Comment:
nit: Can we name these variables `columns` and `number_of_columns` for better readability?